### ISSUE TITLE
Enabling WSL fails during reboot

### ISSUE THREAD

**Issue by gaibo** [Microsoft/WSL#2693]
Windows build number: 10.0.16299.64

Issue: I cannot enable WSL, either through "Enable Features" or by running "Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux" in Admin PowerShell. During the reboot, the screen will flash "unable to something something, undoing changes" and once it boots WSL is still not enabled.

I've tried enabling Developer's Mode, didn't help. I have Ubuntu for WSL installed through Microsoft Store, and when I try to launch that, it tells me to enable WSL, which I apparently can't do.

**Comment by Biswa96** [comment]
Repair your Windows system with dism command. See this article: https://technet.microsoft.com/en-us/library/hh824869.aspx

**Comment by gaibo** [comment]
@Biswa96 No component store corruption detected when running "Dism /Online /Cleanup-Image /ScanHealth" on elevated CMD. Any other suggestions?

**Comment by gaibo** [comment]
@Biswa96 
I finally figured out the issue! No wonder no one else was having it.
Check Point Endpoint Security VPN software for my work was preventing my system from making big changes. I remembered that last time Windows had a major update, it forced me to uninstall the VPN software, and for good reason I guess. The company [Check Point has a history with WSL](http://www.zdnet.com/article/check-points-bogus-windows-subsystem-for-linux-attack/) actually.
We can close this issue now.
